src/twisted/conch/ssh/session.py

```diff
@@ -33,9 +33,10 @@ def __init__(self, *args, **kw):
         self.buf = b''
         self.client = None
         self.session = None
+        self.sessionSetEnv = None
```

Comment:
This interface is already somewhat over-broad; is it really necessary to make `sessionSetEnv` a public read/write variable here?  If so, we definitely need to document it for callers.  I mean, it'd be nice to have documentation of its type either way, but an underscore prefix means that documentation is only really maintainer-facing rather than user-facing ;)